iredis/client.py

```diff
@@ -256,9 +256,9 @@ def send_command(self, raw_command, completer=None):  # noqa
                 return
 
             redis_resp = self.execute(command_name, *args)
-            # if shell, do not render, just run in shell pipe and show the
+            # if shell_command and enable shell, do not render, just run in shell pipe and show the
             # subcommand's stdout/stderr
-            if shell_command:
+            if shell_command and getattr(config, "shell", True):
```

Comment:
Consider add a default attribute to `Config.__init__`(Perfer `None`) instead of dynamic adding attributes.